src/nrc/api/tests/test_scopes.py

```diff
@@ -0,0 +1,344 @@
+from unittest.mock import patch
+from uuid import uuid4
+
+from django.test import override_settings
+
+import requests_mock
+from rest_framework import status
+from rest_framework.test import APITestCase
+from vng_api_common.tests import JWTAuthMixin, get_operation_url, reverse
+
+from nrc.api.choices import SequencetypeChoices
+from nrc.datamodel.choices import ProtocolChoices
+from nrc.datamodel.tests.factories import DomainFactory, SubscriptionFactory
+
+from ..scopes import (
+    SCOPE_DOMAINS_CREATE,
+    SCOPE_DOMAINS_READ,
+    SCOPE_EVENTS_PUBLISH,
+    SCOPE_SUBSCRIPTIONS_CREATE,
+    SCOPE_SUBSCRIPTIONS_DELETE,
+    SCOPE_SUBSCRIPTIONS_READ,
+    SCOPE_SUBSCRIPTIONS_UPDATE,
+)
+
+
+@override_settings(
+    LINK_FETCHER="vng_api_common.mocks.link_fetcher_200",
+    ZDS_CLIENT_CLASS="vng_api_common.mocks.MockClient",
+)
+class ScopeDomains(JWTAuthMixin, APITestCase):
```

Comment:
Normaliter gebruik je in de naam van je `TestCase` ook de naam "Tests" of "TestCase".